gradio/external.py

```diff
@@ -852,9 +852,12 @@ def open_api_stream(
             raise ValueError(
                 f"Invalid file type: {file_type}. Must be 'text_encoded' or 'image'."
             )
-
+    chatbot = gr.Chatbot(
+        allow_tags=True,
```

Comment:
A user should be able to override this if they pass in their own chatbot. So basically we should only do this if kwargs does not include chatbot